### ISSUE TITLE
Wikitude other examples

### ISSUE THREAD

**Issue by afeezaziz** [pbreuss/wikitude-ionic-2-starter-app#5]
Thanks @pbreuss for making this starter project. I am wondering whether will you be interested to expand this repo to actually include all of Wikitude examples? 

Coming from non-JS background, and I learnt Typescript straight away... I think Wikitude own examples are a little hard to follow compared to if I read from Ionic 2 examples.

If you are interested to work on it, I am keen to help. Thanks!

**Comment by pbreuss** [comment]
Hello Afeez,
I thought if I add all Wikitude examples then its more work to remove those not needed, rather than it takes to add those needed. Why exactly would you add all of the examples? When you offer your help, do you mean like regular open source contribution?
Best,
Philipp 

Von meinem iPhone gesendet

**Comment by afeezaziz** [comment]
Hi Philip,

Why? -> Because it would be easier for Ionic 2 newbies, like me, to understand how to use Wikitude with Ionic. I think this is a very interesting plugin and hope that the plugin can be elevated into 'first class' plugin like some of the other plugins available on Ionic Native documentation.

Offer for help -> Yes, something like documenting, finding bugs/problems.
